apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx

```diff
@@ -94,14 +105,31 @@ export default class DetailViewContents extends React.Component {
       dataType: 'json',
       contentType: 'application/json',
       data: JSON.stringify(Object.assign({}, this.state, {response_scores: JSON.stringify(this.state.response_scores)}))
-    }).done(() => {
+    }).done((applicationData) => {
       this.setState({
         editing: false
       });
-      this.props.updateProps({notes: this.state.notes, status: this.state.status});
+      this.props.updateProps({
+        status: applicationData.status,
+        locked: applicationData.locked,
+        notes: applicationData.notes
+      });
     });
   };
 
+  renderLockButton = () => {
+    const statusIsLockable = ApplicationFinalStatuses.includes(this.state.status);
+    return (
+      <Button
+        title={statusIsLockable && `Can only lock if status is one of ${ApplicationFinalStatuses.join(', ')}`}
```

Comment:
Won't this display the title when it is lockable? Shouldn't this be `{!statusIsLockable && ...`